app/src/app/create/navtree/navtree.component.sass

```diff
@@ -29,3 +27,7 @@
   .placeholder.disabled-step
     border-bottom-style: dotted
     border-bottom-color: $disabled-frame-color
+  @media screen and (max-width: 414px)
+    .step
+      span.stepName:not(.current-step span.stepName)
```

Comment:
This selector doesn't work, because `:not()` [accepts only a simple selector](https://drafts.csswg.org/selectors-3/#sel-not), and a simple selector [cannot contain a combinator](https://drafts.csswg.org/selectors-3/#sel-not).